ci-operator/step-registry/openshift/upgrade/qe/test/disconnected/openshift-upgrade-qe-test-disconnected-chain.yaml

```diff
@@ -9,6 +9,7 @@ chain:
   - ref: cucushift-upgrade-mirror-images-by-oc-adm
   - ref: cucushift-upgrade-healthcheck
   - ref: cucushift-upgrade-toimage
+  - ref: enable-qe-catalogsource
```

Comment:
@yunjiang29  Shall we call enable-disconnect-qe-catalogsource in test-disconnected-chain? How we distinguish C2S clusters?

Reply:
I guess so. The `enable-qe-catalogsource-disconnected` step will check if the cluster is the `C2S` one, here: https://github.com/openshift/release/blob/master/ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh#L380

Reply:
> I guess so. The `enable-qe-catalogsource-disconnected` step will check if the cluster is the `C2S` one, here: https://github.com/openshift/release/blob/master/ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh#L380

Right.

Reply:
`enable-qe-catalogsource-disconnected` should be used